README.md

```diff
@@ -23,6 +23,21 @@ You can watch Nodemon do it’s thing if you run `docker logs -f selfservice`.
 
 If you’re making changes to client-side JS or Sass files (anything within [`/browsered/`](https://github.com/alphagov/pay-selfservice/tree/BAU-update-README-to-explain-livereload/app/browsered) or [`/assets/`](https://github.com/alphagov/pay-selfservice/tree/BAU-update-README-to-explain-livereload/app/assets)) then running `npm run watch-live-reload` will watch for changes and recompile. Nodemon does not do anything here as that’s not necessary. If you install the [livereload browser plugin](http://livereload.com/extensions/) then it will refresh your page once the assets have been compiled to the `/public` folder.
 
+### Steps to run locally - new method
+The new method is faster and easier to run in debug mode.
+* `./scripts/generate-dev-environment.sh local` - this will create a local `.env` file.
+* Repeat the same steps above.
+* Run the app with: `npm run start:dev`
+* Go to: http://localhost:3000 -> Note the diff port number.
+
+#### Run in debug mode
```

Comment:
Now that we've added configurations to launch individual mocha tests in debug mode it would be great to document that behaviour. 

I would think about organising all of these under a 'VSCode' heading as they're specific to that IDE